RPIPs/RPIP-43.md

```diff
@@ -32,7 +32,7 @@ This specification introduces a type of smart contract called a "megapool" that
 SHALL serve as the target of beacon chain withdrawal credentials for Rocket Pool
 validators deposited after this RPIP is implemented.
 
-- A node operator SHALL be able to deploy at most one megapool contract
+- A node operator SHALL be able to deploy at most one megapool contract per node
```

Comment:
Technically we don't have visibility if the same node operator opens multiple megapool contracts on different nodes, so should this more clearly state "per node", or "A node SHALL be able to..."

Maybe that muddies the descriptions everywhere else though and this should be ignored

Reply:
I think your change is more correct. I'm unsure on the knock-on effects of changing just here.

Reply:
Yeah, I think this is consistent with how we use NO elsewhere. As we use it, it's more like "node's operator". I prefer not making this change.